packages/nuxt/src/plugins/plugin.ts

```diff
@@ -155,10 +162,13 @@ function buildTarget(
   return targetConfig;
 }
 
-function serveTarget() {
-  const targetConfig: TargetConfiguration<Partial<NuxtServeExecutorOptions>> = {
-    executor: '@nx/nuxt:serve',
-    defaultConfiguration: 'development',
+function serveTarget(projectRoot: string) {
+  const targetConfig: TargetConfiguration = {
+    executor: 'nx:run-commands',
+    options: {
+      command: 'npx nuxi dev',
```

Comment:
You should use `detectPackageManager` + `getPackageManagerCommand` instead of hardcoded `npx`. This would fail with `PnP`